src/Dist_Archive_Command.php

```diff
@@ -184,21 +158,15 @@ public function __invoke( $args, $assoc_args ) {
 		}
 
 		if ( $archive_base !== $source_base || $this->is_path_contains_symlink( $path ) ) {
-			$tmp_dir  = sys_get_temp_dir() . DIRECTORY_SEPARATOR . $archive_base . '.' . $version . '.' . time();
+			$tmp_dir  = sys_get_temp_dir() . DIRECTORY_SEPARATOR . uniqid( $archive_base . '.' . $version );
 			$new_path = $tmp_dir . DIRECTORY_SEPARATOR . $archive_base;
 			mkdir( $new_path, 0777, true );
-			$iterator = new RecursiveIteratorIterator(
-				new RecursiveDirectoryIterator( $path, RecursiveDirectoryIterator::SKIP_DOTS ),
-				RecursiveIteratorIterator::SELF_FIRST
-			);
-			foreach ( $iterator as $item ) {
-				if ( $this->is_ignored_file( $iterator->getSubPathName(), $maybe_ignored_files ) ) {
-					continue;
-				}
-				if ( $item->isDir() ) {
-					mkdir( $new_path . DIRECTORY_SEPARATOR . $iterator->getSubPathName() );
+			foreach ( $this->get_file_list( $path ) as $relative_filepath ) {
+				$source_item = $path . $relative_filepath;
+				if ( is_dir( $source_item ) ) {
+					mkdir( $new_path . '/' . $relative_filepath, 0777, true );
```

Comment:
WP-CLI code style prefers the readability characteristics of string interpolation when applicable:
```suggestion
					mkdir( "{$new_path}/{$relative_filepath}", 0777, true );
```